complete readme redesign (#7530)
denoland/deno: README.md

# Deno

[![Build Status - Cirrus][]][Build status] [![Twitter handle][]][Twitter badge]

<img align="right" src=https://deno.land/logo.svg height="150px">

Deno is a *simple*, *modern* and *secure* runtime for **JavaScript** and **TypeScript** that uses V8 and is built in Rust.

> ### Features

Reviewer: ```suggestion
## Features
```

- Secure by default. No file, network, or environment access, unless explicitly enabled.
- Supports TypeScript out of the box.
- Ships only a single executable file.
- Built-in utilities like a dependency inspector (deno info) and a code formatter (deno fmt).

> - Set of reviewed standard modules that are guaranteed to work with [Deno](https://deno.land/std/)

Reviewer 2: ```suggestion
- Set of reviewed standard modules that are guaranteed to work with [Deno](https://deno.land/std/).
```

> ### Install

Reviewer: ```suggestion
## Install
```

Shell (Mac, Linux):

```sh
curl -fsSL https://deno.land/x/install/install.sh | sh
```

PowerShell (Windows):

```powershell
iwr https://deno.land/x/install/install.ps1 -useb | iex
```

[Homebrew](https://formulae.brew.sh/formula/deno) (Mac):

```sh
brew install deno
```

[Chocolatey](https://chocolatey.org/packages/deno) (Windows):

```powershell
choco install deno
```

Build and install from source using [Cargo](https://crates.io/crates/deno):
```sh
cargo install deno
```

See  [deno_install](https://github.com/denoland/deno_install/blob/master/README.md) and [releases](https://github.com/denoland/deno/releases) for other options.

> ### Getting Started

Reviewer: ```suggestion
## Getting Started
```

Try running a simple program:

```sh
deno run https://deno.land/std/examples/welcome.ts
```

Or a more complex one:

> ```sh

Reviewer 3: I think this `sh` should be `js`.

import { serve } from "https://deno.land/std@0.69.0/http/server.ts";
const s = serve({ port: 8000 });
console.log("http://localhost:8000/");
for await (const req of s) {
  req.respond({ body: "Hello World\n" });
}
```

You can find a more in depth introduction, examples, and environment setup guides in the [manual](https://deno.land/manual).

> More in-depth info can be found in the runtime [documentation](doc.deno.land)

Reviewer: ```suggestion
More in-depth info can be found in the [runtime documentation](https://doc.deno.land/builtin/stable)
```

Reviewer: missing url

Reviewer 2: ```suggestion
More info can be found in the [runtime documentation](https://doc.deno.land/builtin/stable).
```

### Contributing

We appreciate your help!

> To contribute, please read the our [guidelines](https://github.com/denoland/deno/blob/master/docs/contributing/style_guide.md)

Reviewer: ```suggestion
To contribute, please read the our [guidelines](https://deno.land/manual/contributing)
```

Reviewer: I think we should use deno.land/manual instead of a markdown file

Reviewer 2: ```suggestion
To contribute, please read our [guidelines](https://deno.land/manual/contributing).
```
